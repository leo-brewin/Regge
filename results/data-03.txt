# defect
 8.622410782228104180E-03

# i  j   d(defect)/d(lsq(i,j))
  1  2  2.066008164666802600E+01
  1  3  2.069570992550337600E+01
  2  3  2.069608905174731190E+01
  4  1 -2.498195385358791430E+01
  4  2 -2.498127088724986610E+01
  4  3 -2.500067409031011170E+01
  5  1 -3.023495228827060390E+00
  5  2 -3.024328012675571250E+00
  5  3 -3.059111653817606760E+00
  4  5  1.947015169503553850E+01
  6  1 -3.019456888886439230E+00
  6  2 -3.021441504436362670E+00
  6  3 -3.030019371032002900E+00
  5  6  1.941400920895706180E+01
  7  1 -3.039318707529659160E+00
  7  2 -3.036498055419258420E+00
  7  3 -3.022741602341849040E+00
  6  7  1.943695675632065650E+01
  8  1 -3.037095984190230300E+00
  8  2 -3.037930311578392000E+00
  8  3 -3.039344904312957010E+00
  7  8  1.944668217704024180E+01
  9  1 -2.491417017159904150E+01
  9  2 -2.491516560840172820E+01
  9  3 -2.497162693084465490E+01
  8  9  1.947382919508435300E+01
  9  4  3.624125726855521770E+01
