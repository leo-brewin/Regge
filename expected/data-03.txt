# defect
 8.622410782228112150E-03

# i  j   d(defect)/d(lsq(i,j))
  1  2  2.066008164666802580E+01
  1  3  2.069570992550337580E+01
  2  3  2.069608905174731170E+01
  4  1 -2.498195385358791400E+01
  4  2 -2.498127088724986580E+01
  4  3 -2.500067409031011150E+01
  5  1 -3.023495228827060400E+00
  5  2 -3.024328012675571250E+00
  5  3 -3.059111653817606780E+00
  4  5  1.947015169503553860E+01
  6  1 -3.019456888886439210E+00
  6  2 -3.021441504436362680E+00
  6  3 -3.030019371032002920E+00
  5  6  1.941400920895706180E+01
  7  1 -3.039318707529659100E+00
  7  2 -3.036498055419258410E+00
  7  3 -3.022741602341849060E+00
  6  7  1.943695675632065660E+01
  8  1 -3.037095984190230310E+00
  8  2 -3.037930311578392020E+00
  8  3 -3.039344904312956960E+00
  7  8  1.944668217704024180E+01
  9  1 -2.491417017159904110E+01
  9  2 -2.491516560840172780E+01
  9  3 -2.497162693084465450E+01
  8  9  1.947382919508435300E+01
  9  4  3.624125726855521720E+01
